src/main/java/org/threadly/concurrent/benchmark/AbstractSchedulerBenchmark.java

```diff
@@ -10,19 +10,20 @@
 import org.threadly.util.debug.Profiler;
 
 public abstract class AbstractSchedulerBenchmark extends AbstractBenchmark {
-  protected static final int RUNNABLE_COUNT = 1000;
-  protected static final int RUNNABLE_ADD_TIME = 1000 * 5 * 1;
-  protected static final int THREAD_RUN_TIME = 20;
+  protected static final int RUNNABLE_COUNT = 10000;
+  protected static final int RUNNABLE_PER_COUNTER = 2; // higher numbers use less ram but may have contention
+  protected static final int RUNNABLE_ADD_TIME = 1000 * 5;
+  protected static final int THREAD_RUN_TIME = 2;
   protected static final boolean RUN_PROFILER = false;
   
-  protected final AtomicIntegerArray countArray = new AtomicIntegerArray(RUNNABLE_COUNT);
+  protected final AtomicIntegerArray countArray = new AtomicIntegerArray(RUNNABLE_COUNT / RUNNABLE_PER_COUNTER);
```

Comment:
This was done to help facilitate larger tasks counts.  I suspect that anywhere from 2 to 10 tasks per AtomicInteger thread contention will be so low it wont be an issue.  If a _little_ thread contention happens it's fine, it just means we will do less simulated work later on.